### PR TITLE
AP-2132 merits task list

### PR DESCRIPTION
**Pull request by RoseSAK**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-2132)

Created controller and view for merits task list
Call to Legal Framework API in the controller 
Updated flow to redirect to merits task list if multiple proceedings flag is switched on, but continue along current merits path if not 
Added rspec tests for MeritsTaskListsController and updated tests for SerializableMeritsTasksLIsts

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by stephenrichards — DISMISSED** [review]

File: app/controllers/providers/merits_task_lists_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  class MeritsTaskListsController < ProviderBaseController
+    def show
+      @merits_tasks = merits_tasks
+    end
+
+    private
+
+    def merits_tasks
+      YAML.load(LegalFramework::MeritsTasksService.call(@legal_aid_application).serialized_data).tasks # rubocop:disable Security/YAMLLoad:
+    end
```
[the remark this answers is not in the document]
Comment (Obsiye) [reply app/controllers/providers/merits_task_lists_controller.rb:11]: Ah that's my bad. ^ Told her to use yaml.load :grimacing:

**Review by stephenrichards — APPROVED** [review]

File: app/webpack/stylesheets/task-list.scss [hunk 2]
```diff
@@ -4,72 +4,74 @@
...
+  }
 }
 
-.app-task-list__task-completed {
-  float: right;
+.app-task-list__task-name p {
+  margin-bottom: 0;
 }
+
+// The `app-task-list__task-completed` class was previously used on the task
```
Comment (willc-work) [inline app/webpack/stylesheets/task-list.scss:64]: when you say` existing prototypes` do you mean existing deployments in UAT or is this related to the prototypes on heroku? If the latter, is it still required to be included?

File: config/locales/en/providers.yml [hunk 3]
```diff
@@ -444,6 +444,30 @@ en:
...
+        states:
+          not_started: Not started
+          waiting_for_dependency: Cannot start yet
+          complete: Completed
+        urls:
+          latest_incident_details: date_client_told_incident
+          opponent_details: opponent
+          statement_of_case: statement_of_case
+          chances_of_success: chances_of_success_index
+          application_children: chances_of_success_index
```
Comment (willc-work) [inline config/locales/en/providers.yml:469]: are these 3 keys supposed to have the same values?

**Review by willc-work — CHANGES_REQUESTED** [review]
The changes to the `values-uat/staging/production.yaml` files need to be removed or rebased against master. The sentry keys that these files are using are the old ones which were changed today.

File: config/locales/en/providers.yml (hunk 3 above)
Comment (RoseSAK) [reply config/locales/en/providers.yml:469]: Yeah for now, they'll be updated when the other controllers and views for these pages are merged in but they don't exist yet on this branch

File: app/webpack/stylesheets/task-list.scss (hunk 2 above)
Comment (RoseSAK) [reply app/webpack/stylesheets/task-list.scss:64]: this scss is mostly just copied from the design system task list component (which is experimental - also some of it didn't work so I've updated it hence why it looks a bit different if you do compare to their scss) that comment was part of it. I guess they're just saying it's to avoid breaking existing task lists. Yeah I can probably remove this actually
